### PR TITLE
Update thor for rails 6

### PR DESCRIPTION
**Pull request by shishirsharma**
* This is needed for rails6 upgrade.

### CODE REVIEW

File: qless.gemspec [hunk 1]
```diff
@@ -43,7 +43,7 @@ language-specific extension will also remain up to date.
   s.add_dependency 'sinatra', ['>= 1.3', '< 2.1']
   s.add_dependency 'statsd-ruby', '~> 1.3'
   s.add_dependency 'thin', '~> 1.7'
-  s.add_dependency 'thor', '~> 0.19.1'
+  s.add_dependency 'thor', '~> 0.20.3'
```
Comment (bslobodin) [inline qless.gemspec]: https://github.com/erikhuda/thor/compare/v0.19.1...v0.20.3
https://github.com/erikhuda/thor/blob/master/CHANGELOG.md#0203

**Comment by Andreis13** [comment]
it seems there are no updates in `Gemfile.lock`, did you run `bundle install` after bumping the version?

File: spec/integration/job_spec.rb [hunk 2]
```diff
@@ -169,7 +169,7 @@ class NoPerformJob; end
       queue.put('Foo', {}, jid: 'c', depends: ['a'])
       expect(client.jobs['c'].dependencies).to eq(['a'])
       client.jobs['c'].depend('b')
-      expect(client.jobs['c'].dependencies).to eq(%w{a b})
+      expect(client.jobs['c'].dependencies).to match_array(%w{a b})
```
Comment (shishirsharma) [inline spec/integration/job_spec.rb]: Was failing before my changes. Looks like the order of dependacis my not be guaranteed.

**Comment by shishirsharma** [comment]
@Andreis13 Updated. 

With ruby changes as well.

**Comment by sunblaze** [comment]
Got the okay to merge from @shishirsharma we're looking to upgrade Bourgeois to Rails 6

**Pull request merged**